client/app/components/about-modal/about-modal.controller.js

```diff
@@ -0,0 +1,35 @@
+(function() {
+  'use strict';
+    
+  angular.module('app.components')
+    .directive('aboutModal', function() {
+      return {
+        templateUrl: "app/components/about-modal/about-modal.html",
+        controller: ModalController,
+        controllerAs: "vm",
+      };
+    });
+  /** @ngInject */
+  function ModalController(ServerInfo) {
+    var vm = this;
+    vm.additionalInfo = "";
+    vm.copyright = __("Copyright (c) 2016 ManageIQ. Sponsored by Red Hat Inc.");
+    vm.imgAlt = "ManageIQ logo";
```

Comment:
This feels like it will break skinning.  Can you please verify that these strings are changed when productized with CloudForms?